src/assets/stylesheets/components/site/progress/_header-progress.scss

```diff
@@ -0,0 +1,18 @@
+.header-progress-wrapper {
```

Comment:
styles specific to the header progress bar, this checkpoints container could be extracted and put into the linear progress component if we were to reuse it.